content/spin/designing-spin-apps.md

```diff
@@ -0,0 +1,44 @@
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/spin/designing-spin-apps.md"
+
+---
+
+- [Built With Spin](#built-with-spin)
+- [Next Steps](#next-steps)
+
+## Built With Spin
+
+Spin can be used to build many different types of applications. The following blog articles show how different applications are built with Spin:
+
+[How I Built a Like Button for My Blog with Spin](https://www.fermyon.com/blog/how-i-built-a-like-button-for-my-blog-with-spin)
+
+The 'Building a Social App with Spin' series covers the process and decision-making of building an authenticated, dynamic, database-backed application and API, right the way from downloading Spin and setting up the project to deploying the finished application to the cloud.
+
+{{ details "Building a Social App With Spin" "- [Part 1: Project Setup and first API](https://www.fermyon.com/blog/building-a-social-app-with-spin-1)\n- [Part 2: Vue.js App and Token Verification](https://www.fermyon.com/blog/building-a-social-app-with-spin-2)\n- [Part 3: Post API and Spin SDK Bindings](https://www.fermyon.com/blog/building-a-social-app-with-spin-3)\n- [Part 4: Go Postgres Usage](https://www.fermyon.com/blog/building-a-social-app-with-spin-3-5)\n- [Part 5: Key-Value Storage and Fermyon Cloud](https://www.fermyon.com/blog/building-a-social-app-with-spin-4)"}}
+
+[Serving Static Content via WebAssembly](https://www.fermyon.com/blog/serving-static-content-via-webassembly)
+
+[Bots With Spin and Fermyon Cloud](https://www.fermyon.com/blog/bots-with-spin-and-fermyon-cloud)
+
+[Shortlink and QR Code Generator](https://www.fermyon.com/blog/component-reuse)
+
+[Build Your Own Content Management System (CMS) From a Template](https://www.fermyon.com/blog/build-you-own-cms-from-a-template)
+
+The 'Finicky Whiskers' series covers the journey of creating a game with an architecture designed to [re-think microservices](https://www.fermyon.com/blog/rethinking-microservices) and showcase how WebAssembly modules can be started, executed, and shut down in the blink of an eye.
+
+> Finicky Whiskers was built to be the world's most adorable manual load generator, so it's not necessarily an model of how to use Spin most efficiently! But the series may be useful for decomposing an application into Spin microservices, and as a look at how Spin and containers can work together. As a bonus, if you are interested, you can play the game [here](https://www.finickywhiskers.com/index.html).
+
+{{ details "Finicky Whiskers" "- [Part 1: The World's Most Adorable Manual Load Generator](https://www.fermyon.com/blog/finicky-whiskers-part-1-intro)\n- [Part 2: Serving the HTML, CSS, and static assets](https://www.fermyon.com/blog/finicky-whiskers-part-2-fileserver)\n- [Part 3: The Microservices](https://www.fermyon.com/blog/finicky-whiskers-part-3-microservices)\n- [Part 4: Spin, Containers, Nomad, and Infrastructure](https://www.fermyon.com/blog/finicky-whiskers-part-4-infrastructure)"}}
```

Comment:
Same comment re bullets instead of collapsed.

Reply:
Updated in the next push. Thanks for providing an example for the previous example.